spec/ddtrace/contrib/rails/redis_cache_spec.rb

```diff
@@ -40,7 +40,12 @@
   end
 
   let(:cache_store_name) do
-    Gem.loaded_specs['redis-activesupport'] ? 'redis_store' : 'redis_cache_store'
+    if Gem.loaded_specs['redis-activesupport'] \
```

Comment:
this change seems unrelated?

Reply:
No, tests were breaking. No idea how these were passing before but we changed the default store for Rails 5+ to use the internal Redis implementation instead of the gem.

Reply:
hmm, weird if it is only failing on this branch, otherwise if unrelated to your changes it should probably be in a separate PR